tests/integration/workflow.py

```diff
@@ -68,6 +68,19 @@ def cancel_workflow():
     workflow_execution.request_cancel()
 
 
+@activity.with_attributes(task_list="quickstart", version="example", idempotent=True)
+def pingpong_activity(d: str, n: float | int) -> str:
+    return pingpong(d, n)
+
+
+def pingpong(d, n):
+    if n == 1:
+        return d
+    if n < 1:
+        return d[0 : int(len(d) * n)]
+    return d * n
```

Comment:
n is expected to be a float is less than one, but has to be an int if more than one. That strange.

Plus, the name `d` and `n` (`n` being a float is strange)

Also, the name of the function is cryptic.